LICENSE.txt

```diff
@@ -0,0 +1,28 @@
+BSD 3-Clause License
+
+Copyright (c) 2023, José Rio
```

Comment:
Shouldn't here be OutSystems, instead of José Rio?

Reply:
Done! Tks Bernardo!